src/components/CategoryPicker.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+export default function CategoryPicker(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <div className={classes.paper}>
+        <Typography component="h2" variant="h6">
+          Search with a Relationship
+        </Typography>
+        <Typography>
+          Hold shift while clicking to select more than one category to include{' '}
```

Comment:
Is the {' '} necessary? I can't tell just from looking at the code. :-)

Reply:
Sorry! I thought I pushed the updated version with the drop down box, The 'hold shift while...' was removed. But you are right! {' '} is not necessary but when I save that pops up, I will have to be more aware of that in the future and remove it!